src/routes.jsx

```diff
@@ -0,0 +1,23 @@
+import { Route, Routes } from 'react-router-dom'
+import { MainPage } from './components/Mainpage'
+import { Baklazhan } from './components/Baklazhan/Baklazhan'
+import { Veranda } from './components/Veranda'
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage'
+import { Wishlist } from './components/Wishlist/Wishlist'
```

Comment:
Обращай внимание, чтобы из index тащился компонент
```suggestion
import { Wishlist } from './components/Wishlist'
```